libtransact/src/workload/runner.rs

```diff
@@ -338,7 +341,18 @@ impl WorkerBuilder {
 
                                 // get next target, round robin
                                 next_target = (next_target + 1) % targets.len();
-                                thread::sleep(time_to_wait);
+                                let diff = time::Instant::now() - submission_start;
+                                let submission_time = match submission_avg {
+                                    Some(val) => {
```

Comment:
This looks like it reduces to avg=(diff+avg)/2; suggest we simplify the implementation here until we need a more complex approach.

Reply:
Simplified to just that expression